report/main.tex

```diff
@@ -0,0 +1,810 @@
+The supervisor node also keeps track of some useful variables; namely the race type, start and end times, lap count, average speed, and battery life. When the race is finished, a few of these variables are stored and saved into a text file. These variables are used to make important decisions regarding when to start and stop the race. The race is started when a traffic light signal is received, and is ended when either the lap count is reached for that race (1 for drag, 3 for circuit), which is incremented by an end line detection signal, or if the battery life dips below 10%.
+
+\subsection{Computer Vision}
+This competition year huge strides we're made in improving the robustness and functionality of our computer vision software. To improve the robustness of our lane detection system a shadow removal algorithm was added as a pre-processing step on each of our cameras. To improve our traffic light detection in all conditions a machine learning classifier was added to help detect traffic lights in images. End line line detection was improved considerable over past years as well largely due to the addition of a new hysteresis filter on the camera stream. 
+
+\subsubsection{Shadow Removal}
+In last year’s competition, the robot had difficulty in reliably tracking lane lines under variable lighting conditions, such as when buildings or robots cast shadows onto the track. As a result, this year’s team implemented a shadow detection and removal pre-processing step, based on work done by Murali et al[cite], in order to address this issue.
+First, to detect shadows, the algorithm calculates the mean value of each plane in the Lab colour space and applies a threshold on the L and b values. The Lab colour space was designed to be uniform with respect to human vision, so it could better model visually perceived changes, such as variations in lightness that would indicate a shadow region. An example is shown in Figure X1.
```

Comment:
I referred to "Figure X_" in my write-up because I didn't know what figure numbers they would be. These should be updated (never used Latex before, maybe they can be linked somehow?).